### PR TITLE
Add Date/akamai-time-reference

### PR DESCRIPTION
**Pull request by jucrouzet**
Gives a promise of a Dateobject with the reference actual date/time using Akamai's Time Reference Server.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 > A curated list of delightful Node.js [packages](#packages) and [resources](#resources).
 
 *Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing. You might also like [awesome-npm](https://github.com/sindresorhus/awesome-npm).*
-
+Da
```
Comment (sindresorhus) [inline readme.md]: ?

**Comment by sindresorhus** [comment]
Please read and adhere to the [contributing guidelines](../blob/master/contributing.md).

**Comment by sindresorhus** [comment]
It's unclear what the use-case is for this module?

**Comment by jucrouzet** [comment]
_(As english is not my native language, sorry if I'm not clear)_

**Comment by sindresorhus** [comment]
:palm_tree: Thanks for the suggestion, but I'm going to pass on this :)

**Pull request closed without merging**